sqldelight-idea-plugin/testData/missing-where/src/Example.sq

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE example (
+  id INTEGER NOT NULL PRIMARY KEY,
+  name TEXT NOT NULL
+);
+
+DELETE FROM example WHERE id = 1;
+
+DELETE FROM example;
```

Comment:
Not sure how I feel about this being linted. I have this for almost every table to prune it.

Reply:
same for me, I think in general its a reasonable type of query to run

Reply:
if theres a way to have it default off i think id b cool with that

Reply:
This is similar to https://www.jetbrains.com/help/phpstorm/sql-delete-or-update-statement-without-where-clauses.html and it's enabled by default

Anyway, I'm going to turn it off